File: Web/src/app/core/models/feed-log.ts
import { TemplateRef } from "@angular/core"

export interface FeedTrackerData {
  bottleSize: number
  feedType: string
  breastSide: string[]
  milkType: string[]
  milkConsumed: number
  startTime: string
  endTime: string
  notes: string
}

export interface FeedStep {
  name: string
  templateRef?: TemplateRef<any>
}

File: Api/src/MilkAndMoon.Api/Models/FeedLog.cs
public class FeedLog : BaseLog
{
    public decimal BottleSize { get; set; }
    public string FeedType { get; set; } = null!;
    public string[] BreastSide { get; set; } = null!;
    public string[] MilkType { get; set; } = null!;
    public decimal MilkConsumed { get; set; }
    public DateTimeOffset? EndTime { get; set; }
}
